buildmaster/LHCB_Z_13TEV_DIMUON_2022/metadata.yaml

```
@@ -0,0 +1,104 @@
+setname: "LHCB_Z_13TEV_DIMUON_2022"
```

```suggestion
setname: "LHCB_Z_13TEV_DIMUON_2022"
```

This can be either `Z0` (for Z production) or `Z0J` (for Z + J production).
you cannot have both processes in the same dataset even if they are coming from the same paper. Please break it down into two distinct datasets.

Also, I'd avoid using `2022` in the name of the dataset unless it is necessary to disambiguate.

---

The paper deals with Z production only. So I'll change it to `Z0`.

The `2022`label I thought was sort of necessary to tell it apart from the older `LHCB_Z0` datasets which date back to 2016 and that are already implemented.

---

ZpT is Z + jet, that's why we have those as Z0J

---

Yes, but legacy datasets fro the same experiment are `Z0`. See for instance [LHCB_Z0_13TEV](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/datafiles/new_commondata/LHCB_Z0_13TEV/metadata.yaml). So I suppose I should do the same here, shouldn't I?

Moreover, since the same name is used for the implementation linked above, should I keep the `2022` label?

---

With respect to the `2022` label, I guess we can live with that. But I'd prefer something that includes some information (like for instance the integrated luminosity, since is what we have in a few places already `XYPB`).

With respect to the process names, at the moment we have no Z+J process in for `LHCB` so this would be the first one.

Note that the process label `Z0` (or `Z0J`) refers in practice to the leading order process being considered.
For the Z rapidity this corresponds to $p \ p -> Z -> l^{+} \ l^{-}$ which we are calling `Z0` while for the Z pt is instead $p \ p -> Z + j$ where the measurement is the pT of the jet (and, if double differencial, pt and rapidity) and $j$ is a jet, and we are calling this one `Z0J`.

The difference is important since, for instance, this corresponds to distinct processes when computing predictions with matrix / nnlojet / madgraph.

---

Ok, now I understand. It wasn't clear to me that `Z0` and `Z0J` must be broken down into two different datasets (i.e. two different folders), even if they come from the same measurement. I would suggest to specify that in the [documentation](https://docs.nnpdf.science/data/commondata.html), because it's not trivial for a novice (as I am).

RE the `2022` label, I can change it to the integrated luminosity if you wish. Note that the legacy implementation for `Z0` does not report this information in the name.